pkg/util/k8s_register.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
we live in 2019 :-)

Reply:
This is taken from  https://github.com/kubernetes-csi/cluster-driver-registrar/blob/master/cmd/csi-cluster-driver-registrar/k8s_register.go#L2 do i still need to change year?